epyqlib/variableselectionmodel.py

```diff
@@ -787,7 +789,8 @@ def parse_log(self, data, csv_path):
             [hash_node] = [
                 n for n in block_header_node.children if n.fields.name == "softwareHash"
             ]
-            if self.git_hash != hash_node.fields.value:
+            log_hash = f"{hash_node.fields.value:07x}"
```

Comment:
What happens when `hash_node.fields.value == None`? Line #794 indicates that it is possible for that value to be `None`.

Reply:
Quite so.  Hmm...  Were you able to actually make code that makes that happen without bypassing the code that actually fills out the values?  I can't say I've got it all in my head but I'm not clear how we end up with a `None` there.

But whatever, in favor of a minimal change I'll adjust to account for this previously-handled case.

Reply:
I just went by what I saw in the code, which was that it was possible `hash_node.fields.value` could be `None`. I snipped the section of code out that you had changed and ran tests on that. No idea if `hash_node.fields.value == None` could happen in reality, but the code suggests it's possible.
Your adjustment looks good and passes my ad-hoc tests.